spec/index.bs

```diff
@@ -2179,12 +2211,83 @@ interface GPUTextureUsage {
+                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be less than or equal to
+                                        |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be less than
+                                        or equal to |this|.{{GPUAdapterLimits/maxTextureDimension3D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be 1.
+                                    :: |descriptor|.{{GPUTextureDescriptor/format}} must not be a compressed format or depth/stencil format.
+                                </dl>
+                            - If |descriptor|.{{GPUTextureDescriptor/sampleCount}} > 1 (|descriptor|.{{GPUTextureDescriptor/sampleCount}} is 4):
+                                - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be 1.
+                                - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be 1.
+                                - |descriptor|.{{GPUTextureDescriptor/usage}} must not include the {{GPUTextureUsage/STORAGE}} bit.
+                                - |descriptor|.{{GPUTextureDescriptor/format}} must be a [=renderable format=].
+                            - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be less than or equal to
+                                [$maximum mipLevel count$](|descriptor|.{{GPUTextureDescriptor/dimension}}, |descriptor|.{{GPUTextureDescriptor/size}}).
+                            - |descriptor|.{{GPUTextureDescriptor/usage}} must be a combination of {{GPUTextureUsage}} values.
+                            - If |descriptor|.{{GPUTextureDescriptor/usage}} includes the {{GPUTextureUsage/RENDER_ATTACHMENT}} bit,
+                                |descriptor|.{{GPUTextureDescriptor/format}} must be a [=renderable format=].
+                            - If |descriptor|.{{GPUTextureDescriptor/usage}} includes the {{GPUTextureUsage/STORAGE}} bit,
+                                |descriptor|.{{GPUTextureDescriptor/format}} must be listed in [[#plain-color-formats]] table
+                                with {{GPUTextureUsage/STORAGE}} capability.
+                            - Some |descriptor|.{{GPUTextureDescriptor/format}}s require [=feature=]s when creating a {{GPUDevice}}:
+                                - {{GPUTextureFormat/"depth24unorm-stencil8"}} requires [[#depth24unorm-stencil8]].
+                                - {{GPUTextureFormat/"depth32float-stencil8"}} requires [[#depth32float-stencil8]].
+                                - BC compressed format requires [[#texture-compression-bc]].
```

Comment:
I'd prefer to express this info in tables instead of duplicating it here (and having to always update it here). For compressed textures, it's already written in [the table](https://gpuweb.github.io/gpuweb/#packed-formats). We should add it to the depth/stencil formats table as well, and here just say, if the format table says it requires a feature, the feature must be enabled. See steps 1+2 of https://gpuweb.github.io/gpuweb/#dom-gpudevice-createqueryset for example.

Also note, usage of an unenabled feature enum should throw an exception, rather than failing validation, per https://github.com/gpuweb/gpuweb/blob/main/design/ErrorConventions.md

Reply:
Done. But I don't point to depth/stencil format table because it is for mainly for copy. I point to `GPUTextureFormat` which also list the feature required via comment. 

Good to know that it should be an exception if the required feature is not included. Thanks!